vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/VLogReasoner.java

```diff
@@ -808,4 +813,59 @@ ReasonerState getReasonerState() {
 	void setReasonerState(ReasonerState reasonerState) {
 		this.reasonerState = reasonerState;
 	}
+
+	@Override
+	public Correctness writeInferences(OutputStream stream) throws IOException {
+		QueryResult queryAnswer;
+		Set<Predicate> toBeQueriedHeadPredicates = new HashSet<Predicate>();
+		TermQueryResultIterator stringQueryResultIterator;
+		QueryResultIterator answers;

```

Comment:
same as above with `queryAnswer`.

Generally, declare a variable just before you use it, this makes code more readable.